tests/test_datasets.py

```diff
@@ -215,3 +220,101 @@ def test_item():
         )
 
         item = Item(inputs=inputs, outputs=outputs, forcing=forcing)
+
+
+class FakeDataset(DatasetABC):
+    """
+    Instanciate a fake dataset
+    """
+
+    def __init__(self):
+        pass
+
+    def torch_dataloader(self):
+        pass
+
+    def dataset_info(self):
+        pass
+
+    def meshgrid(self):
+        pass
+
+    def geopotential_info(self):
+        pass
+
+    def cache_dir(self):
+        pass
+
+    def from_json(self):
+        pass
+
+
+class FakeGrid:
+    """
+    Instanciate a fake grid
+    """
+
+    def __init__(self, lat, lon):
+        self.lat = lat
+        self.lon = lon
+
```

Comment:
See my first comment: if you decouple the forcing methods from the dataset class, you won't need to create these fake dataset and fake grid.